.github/workflows/ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       matrix:
         include:
           - crate: rune
-            target: "runtime:: hir::arena:: --skip runtime::vm:: --skip runtime::vm_execution::"
+            target: "runtime:: hir::arena:: --skip runtime::vm:: --skip runtime::vm_execution:: --skip runtime::function::"
```

Comment:
This does technically skip one existing test, `assert_send_sync`, but it seems like that test is designed to fail at (test) compile time, so it shouldn't need to be run under miri

Reply:
Why are we skipping over the `runtime::function` tests to begin with?

Reply:
The tests I added there seemed to be taking a very long time to run under miri CI, and I didn't want to clog up the CI build for everyone else. It shouldn't be a problem after I move them to the `tests` folder, though.